app/presenters/user_navigation.rb

```diff
@@ -15,6 +15,10 @@ def items
     @items ||= [] + artist_nav_items + user_nav_items + [link_to('sign out', url_helpers.logout_path)]
   end
 
+  def remind_for_open_studio_register?
+    current_artist&.remind_for_open_studios_register?
```

Comment:
I think this could be 
```
def remind_for_open_studios_register?
  current_event = OpenStudiosEventService.current
  return false unless current_event || current_artist&.doing_open_studios?
  
  current_event.start_date <= Time.zone.now + 12.weeks
end
```
which takes that `remind` method off the artist and moves it here.